security/docker/Dockerfile.node-agent-k8s

```diff
@@ -1,5 +1,4 @@
-FROM ubuntu:16.04
+FROM istionightly/base_debug
 ADD node_agent_k8s /
-RUN apt-get update && apt-get install -y ca-certificates
```

Comment:
i didn't find the nightly base docker file,  we don't need ca-certs, because nightly base image already include them?

Reply:
yes, see https://github.com/istio/istio/pull/10949